src/BattleCalculator-Freewar/BattleCalculator-Freewar.user.js

```diff
@@ -100,8 +105,8 @@ function processElement(cellElement) {
 			$(npcFastAttackElement).removeAttr('onclick');
 			$(npcFastAttackElement).hide();
 			$(npcNameElement).addClass('processedNPC knownNPC');
-		} else if (initIgnoreNpcData(npcName)) {
-			// nothing
+		} else if (initIgnoreNpcData(npcName) && npcAutoAttack == true) {
```

Comment:
u can leave out the `== true`, i.e. just `if (foo)`, not `if (foo == true)`. other than that, cool stuff :)